apps/meeting/src/containers/MeetingRoster.tsx

```diff
@@ -6,7 +6,8 @@ import {
   Roster,
   RosterHeader,
   RosterGroup,
-  useRosterState
+  useRosterState,
+  RosterAttendeeType
```

Comment:
nit: add trailing comma here

Reply:
I think this eslint config removes the trailing comma. I can look on adding that rule.

Reply:
What is the benefit of having trailing comma?

Reply:
One of the benefit is one you try to add/remove one line, the git diff is one-line change, instead of two.
For example:
```
// No trailing comma
{
  RosterGroup,
  useRosterState,
  RosterAttendeeType
}

// A new change requires two-line change
{
  RosterGroup,
  useRosterState,
  RosterAttendeeType,    // <=== add comma here
  NewDependency  // <==== add new line here
}
```

This makes version control diffs cleaner and editing code might be less troublesome. 

Just a best practice like new line end of file.

Reply:
i see. Make sense.  I never remember it. Thank @xuesichao

Reply:
I have updated the code base and added trailing commas rule.